subprojects/common/report-viewer/src/main/kotlin/com/avito/report/TestSuiteNameProvider.kt

```diff
@@ -0,0 +1,11 @@
+package com.avito.report
+
+interface TestSuiteNameProvider {
+
+    fun getName(): String
+
+    object Stub : TestSuiteNameProvider {
```

Comment:
Is it only for testing? Can we put it in a testFixtures source set?

Reply:
Renamed it to NoOp; i don't know how it will be used without RV yet